eth_crowdfund_be/eth_crowdfund_api/src/config.py

```diff
@@ -14,8 +14,15 @@ class Production(object):
     SQLALCHEMY_DATABASE_URI = "postgresql://postgres:password@localhost/eth_crowdfund_api_db"
     JWT_SECRET_KEY = os.getenv('JWT_SECRET_KEY')
 
+class Testing(object):
+  TESTING = True
+  JWT_SECRET_KEY = os.getenv('JWT_SECRET_KEY')
+  SQLALCHEMY_DATABASE_URI = "postgresql://postgres:password@localhost/eth_crowdfund_api_db"
```

Comment:
We should probably set up a testing database 'eth_crowdfund_api_test_db'

Reply:
good thought. I'll change this